include/system_heap.h

```diff
@@ -0,0 +1,9 @@
+#ifndef SYSTEM_HEAP
+#define SYSTEM_HEAP
+
+#include "libc/stddef.h"
+#include "PR/ultratypes.h"
+
+void SystemHeap_Init(void* start, size_t size);
```

Comment:
Seems like this header should contain the other functions that are obviously meant to be external but happen to be unused.

Reply:
tbh, I'm not sure there is anything else that obviously should be exposed. Which other ones do you think should be exposed?

Reply:
Usually you'd expose the malloc and free, although here perhaps it's not very clear that they would be used elsewhere. The block function stuff is weird since it's unused in here, not clear how to use it outside.